posthog/api/user.py

```diff
@@ -97,14 +98,20 @@ def redirect_to_site(request):
         "apiURL": request.build_absolute_uri("/"),
         "userIntent": request.GET.get("userIntent"),
     }
-    if settings.DEBUG:
-        params["jsURL"] = "http://localhost:8234/"
-    if request.user.toolbar_mode == "toolbar":
+
+    if settings.JS_URL:
+        params["jsURL"] = settings.JS_URL
```

Comment:
if DEBUG is not set this will be an empty string, is that correct?

Reply:
Yeah, that's fine. Empty string is falsy in python (based on my quick test), so then it won't set params['jsURL']... just like now it won't set if not in DEBUG mode. The JS counterpart will substitute it for apiURL then.